src/main/kotlin/no/nav/syfo/dialogmote/DialogmoteService.kt

```diff
@@ -143,7 +145,12 @@ class DialogmoteService(
                     varselUuid = motedeltakerArbeidstakerVarselIdPair.second,
                 )
 
-                // TODO: Implement DialogmoteInnkalling-Varsel to Arbeidsgiver
```

Comment:
Kan godt beholde TODO med endring der hvor det skal lages varsler til NL.
Grunnen til det er at MQ-sending til NL er kun en del av "Send varsel til NL"-flyten.

Reply:
Ps: Om vi skal sende med en UUID til `narmestelederService` som er referanse til varselet til Arbeidsgiver, så trenger vi å opprettet en flyt for `MOTEDELTAKER_ARBEIDSGIVER_VARSEL` tilsvarende som det som er gjort for `MOTEDELTAKER_ARBEIDSTAKER_VARSEL`.
Kan jo fjerne uuid-referansen for denne oppgaven sin skyld og så kan de legges til med resterende funksjonalitet.